backend/pom.xml

```diff
@@ -23,6 +23,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Diese Dependencies benötigt es nicht. Du verwendest diese Annotations in deinem Code nicht. Bitte löschen.